jplag.endToEndTesting/src/main/java/de/jplag/end_to_end_testing/modelRecord/Options.java

```diff
@@ -0,0 +1,39 @@
+package de.jplag.end_to_end_testing.modelRecord;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public record Options(@JsonProperty("minimum_token_match") Integer minimumTokenMatch) {
+
+    /**
+     * Compares like inside values with the passed object. is necessary to find the correct results in the deserialized json
+     * file.
+     */
+    @JsonIgnore
+    @Override
+    public boolean equals(Object options) {
+        if (options instanceof Options optionsCaseted) {
+            return minimumTokenMatch == optionsCaseted.minimumTokenMatch();
+        } else {
+            return false;
+        }
+    }
+
+    /**
+     * Creates the hashCode for the AKtuelle Options object
```

Comment:
The what?

Reply:
Delete hashcode. It's not needed for records :)

Reply:
Also delete equals. It's created in a record implicitly

Reply:
And toString as well . This should also be generated implicitly

Reply:
removed the above mentioned functions